### PR TITLE
Added possibility to use CDATA sections in XML files

### PR DESCRIPTION
**Pull request by tobiasstadler**

### CODE REVIEW

**Comment by lordofthejars** [comment]
Thank you very much for your contribution, looks good to me, let's wait for @bartoszmajsak review as well and we merge it.

**Comment by tobiasstadler** [comment]
Any chance for getting this in?

**Comment by lordofthejars** [comment]
Tomorrow morning I ll take a look

**Comment by bartoszmajsak** [comment]
Thanks for your contributions. Would you share some use cases when this is useful? We could use it then in the blog post announcement.

File: spi/src/main/java/org/jboss/shrinkwrap/descriptor/spi/node/Node.java [hunk 1]
```diff
@@ -222,6 +227,34 @@ public void setComment(final boolean comment) throws IllegalArgumentException {
...
+    /**
+     * Marks this {@link Node} as a CDATA section
+     *
+     * @param cdataSection
+     *            Whether or not this is a CDATA section
+     * @throws IllegalArgumentException
+     *             If this node has children
+     */
+    public void setCDATASection(final boolean cdataSection) throws IllegalArgumentException {
+        // Cannot have children
```
Comment (bartoszmajsak) [inline spi/src/main/java/org/jboss/shrinkwrap/descriptor/spi/node/Node.java]: Both comments in this method are not really needed for my taste as the code is self explanatory. Can you remove those?

**Review by bartoszmajsak — CHANGES_REQUESTED** [review]
Just small thing, thanks again!

**Comment by tobiasstadler** [comment]
In my case the CDATA-section prevents my IDEs code formatting from screwing up the yams configuration

**Review by bartoszmajsak — APPROVED** [review]

**Comment by bartoszmajsak** [comment]
Thank you for your contribution @tobiasstadler!

**Pull request merged**